prisma/schema.prisma

```diff
@@ -271,18 +272,18 @@ enum PaymentType {
 }
 
 model Payment {
-  id             Int      @id @default(autoincrement())
-  uid            String   @unique
-  type           PaymentType
-  bookingId      Int
-  booking        Booking?  @relation(fields: [bookingId], references: [id])
-  amount         Int
-  fee            Int
-  currency       String
-  success        Boolean
-  refunded       Boolean
-  data           Json
-  externalId     String   @unique
+  id         Int         @id @default(autoincrement())
+  uid        String      @unique
+  type       PaymentType
+  bookingId  Int
+  booking    Booking?    @relation(fields: [bookingId], references: [id], onDelete: NoAction, onUpdate: Cascade)
```

Comment:
This means that when we delete a booking we don't do anything with the Payment. We probably never want to delete a payment record.

Unsure how this works in practice since I think the `bookingId` will be a foreign key on something that just got deleted which will probably stop the deletion.

Alternative would be a set NULL and making `bookingId` optional.